src/lib/uploader.ts

```diff
@@ -94,6 +99,13 @@ export default class MinidumpUploader {
         // We either succeeded or something went horribly wrong
         // Either way, we can remove the minidump file
         await unlink(request.path);
+
+        // Also remove it from queued minidumps
+        const storedRequests = this.queue.get();
+        storedRequests.splice(storedRequests.indexOf(request), 1);
```

Comment:
This will blow up if the event is somehow not found in the queue. It'll do `splice(-1, 1)`, which removes last element.

I'd go with
```js
this.queue.set((storedRequests) => storedRequests.filter(storedRequest => storedRequest !== request));
```